components/pluralrules/src/rules/resolver.rs

```diff
@@ -49,7 +49,7 @@ fn test_relation(relation: &ast::Relation, operands: &PluralOperands) -> bool {
 fn calculate_expression(expression: &ast::Expression, operands: &PluralOperands) -> Option<u64> {
     let value = match expression.operand {
         ast::Operand::N => {
-            if operands.n.fract() < f64::EPSILON {
+            if operands.n().fract() < f64::EPSILON {
```

Comment:
Can you check `w == 0` here instead?

Reply:
In that case, why have `n` to begin with?

Reply:
Maybe we should just delete it altogether.  @zbraniecki?

Reply:
My rationale for keeping `n` is to:
* conform to UTS #35 
* allow someone to work with examples and test against them
* provide a convenient way to display the number that the operands are for. If we know that it is not used in runtime calculations (which it seems like!) then keeping it as a getter gives us a cheap way to have it.

It also is my understanding that DCE will make it cost nothing in use cases where noone calls it.

I'd prefer to keep it, but I'm open to be convinced to remove it in 0.2 as we learn about the uses

Reply:
OK, let's keep it.

Dead code elimination should work as long as PluralOperands is a struct.  If it becomes a trait object, I think the compiler won't be able to eliminate it via DCE.  That's maybe one reason to keep this as a struct.